src/Core/Services/SyncService.cs

```diff
@@ -365,7 +365,8 @@ private async Task SyncSettingsAsync(string userId, DomainsResponse response)
 
         private async Task SyncPolicies(List<PolicyResponse> response)
         {
-            var policies = response.ToDictionary(p => p.Id, p => new PolicyData(p));
+            var policies = response?.ToDictionary(p => p.Id, p => new PolicyData(p)) ??
+                           new Dictionary<string, PolicyData>();
```

Comment:
indented formatting should just have this 4 spaces over

```suggestion
                new Dictionary<string, PolicyData>();
```